Manager/PasteboardManager.m

```diff
@@ -183,11 +183,12 @@ - (void)updatePasteboardWithItem:(PasteboardItem *)item fromHistoryWithKey:(NSSt
     }
 
     // the given item gets removed, as the pasteboard is updated with the same item
+    [_pasteboard changeCount];
     [self removePasteboardItem:item fromHistoryWithKey:historyKey shouldRemoveImage:YES];
 
     // automatic paste should not occur for asynchronous operations
     if ([self automaticallyPaste] && shouldAutoPaste) {
-        CFNotificationCenterPostNotification(CFNotificationCenterGetDarwinNotifyCenter(), (CFStringRef)kNotificationKeyHelperPaste, nil, nil, YES);
+        CFNotificationCenterPostNotification(CFNotificationCenterGetDarwinNotifyCenter(), (CFStringRef)kNotificationKeyHelperPaste, nil, nil, NO);
```

Comment:
Why make this not deliver immediately?

Reply:
Actually this value is ignored. See: [CFNotificationCenterPostNotification](https://developer.apple.com/documentation/corefoundation/1542592-cfnotificationcenterpostnotifica?language=objc)